src/DotNetBridge/DotNetBridge.csproj

```diff
@@ -31,12 +31,13 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.ML" Version="0.7.0" />
-    <PackageReference Include="Microsoft.ML.CpuMath" Version="0.7.0" />
-    <PackageReference Include="Microsoft.ML.HalLearners" Version="0.7.0" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="0.7.0" />
-    <PackageReference Include="Microsoft.ML.LightGBM" Version="0.7.0" />
-    <PackageReference Include="Microsoft.ML.Onnx" Version="0.7.0" />
-    <PackageReference Include="Microsoft.ML.TensorFlow" Version="0.7.0" />
+    <PackageReference Include="Microsoft.ML" Version="0.10.0-preview-27310-10" />
```

Comment:
>0.10.0-preview-27310-10 [](start = 54, length = 23)

Is this intentional? Shouldn't we use a release version?

Reply:
Yes. 0.9 won't work and we don't have the official 0.10 yet. So we're stuck with preview nugets until 0.10 is released.

---
In reply to: [248377040](https://github.com/Microsoft/NimbusML/pull/77#discussion_r248377040) [](ancestors = 248377040)

Reply:
Also, we won't be publishing a new nimbusml wheel file until we use 0.10.

---
In reply to: [248401711](https://github.com/Microsoft/NimbusML/pull/77#discussion_r248401711) [](ancestors = 248401711,248377040)